Nightly build errors, spadd_symbolic linking errors

Nightly builds (first detected with Trilinos builds with kokkos and kokkos-kernels develop branches) are failing with linking errors following merge of #1399. Snip below is output from cuda/9.2 + gcc/7.2.0 build, though this is occurring across other nightly builds so far

```
FAILED: kokkos-kernels/unit_test/KokkosKernels_sparse_serial.exe 
: && /projects/sems/install/rhel7-x86_64/sems/compiler/gcc/7.2.0/openmpi/4.0.2/bin/mpicxx  -expt-extended-lambda -arch=sm_35  -Xcudafe --diag_suppress=esa_on_defaulted_function_ignored -O3 -DNDEBUG   kokkos-kernels/unit_test/CMakeFiles/KokkosKernels_sparse_serial.dir/Test_Main.cpp.o kokkos-kernels/unit_test/CMakeFiles/KokkosKernels_sparse_serial.dir/serial/Test_Serial_Sparse.cpp.o  -o kokkos-kernels/unit_test/KokkosKernels_sparse_serial.exe  kokkos-kernels/unit_test/libkokkoskernels_gtest.a  kokkos-kernels/src/libkokkoskernels.a  kokkos/algorithms/src/libkokkosalgorithms.a  kokkos/containers/src/libkokkoscontainers.a  kokkos/core/src/libkokkoscore.a  -ldl  /projects/sems/install/rhel7-x86_64/sems/compiler/cuda/9.2/base/lib64/libcudart.so  /projects/sems/install/rhel7-x86_64/sems/compiler/cuda/9.2/base/lib64/libcublas.so  /projects/sems/install/rhel7-x86_64/sems/compiler/cuda/9.2/base/lib64/libcufft.so  /projects/sems/install/rhel7-x86_64/atdm/tpl/openblas/0.3.6/gcc/7.2.0/lib/libopenblas.so  /projects/sems/install/rhel7-x86_64/sems/compiler/cuda/9.2/base/lib64/libcusparse.so && :
kokkos-kernels/unit_test/CMakeFiles/KokkosKernels_sparse_serial.dir/serial/Test_Serial_Sparse.cpp.o: In function `void KokkosSparse::Experimental::spadd_symbolic<KokkosKernels::Experimental::KokkosKernelsHandle<unsigned long, int, double, Kokkos::Serial, Kokkos::HostSpace, Kokkos::HostSpace>, Kokkos::View<unsigned long const*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::Experimental::EmptyViewHooks, Kokkos::MemoryTraits<0u> >, Kokkos::View<int const*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::Experimental::EmptyViewHooks, Kokkos::MemoryTraits<0u> >, Kokkos::View<unsigned long const*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::Experimental::EmptyViewHooks, Kokkos::MemoryTraits<0u> >, Kokkos::View<int const*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::Experimental::EmptyViewHooks, Kokkos::MemoryTraits<0u> >, Kokkos::View<unsigned long*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::Experimental::EmptyViewHooks, Kokkos::MemoryTraits<0u> >, Kokkos::View<int*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::Experimental::EmptyViewHooks, Kokkos::MemoryTraits<0u> > >(KokkosKernels::Experimental::KokkosKernelsHandle<unsigned long, int, double, Kokkos::Serial, Kokkos::HostSpace, Kokkos::HostSpace>*, Kokkos::View<unsigned long const*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::Experimental::EmptyViewHooks, Kokkos::MemoryTraits<0u> >, Kokkos::View<int const*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::Experimental::EmptyViewHooks, Kokkos::MemoryTraits<0u> >, Kokkos::View<unsigned long const*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::Experimental::EmptyViewHooks, Kokkos::MemoryTraits<0u> >, Kokkos::View<int const*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::Experimental::EmptyViewHooks, Kokkos::MemoryTraits<0u> >, Kokkos::View<unsigned long*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::Experimental::EmptyViewHooks, Kokkos::MemoryTraits<0u> >)':
tmpxft_00007a8e_00000000-5_Test_Serial_Sparse.cudafe1.cpp:(.text._ZN12KokkosSparse12Experimental14spadd_symbolicIN13KokkosKernels12Experimental19KokkosKernelsHandleImidN6Kokkos6SerialENS5_9HostSpaceES7_EENS5_4ViewIPKmJNS5_10LayoutLeftENS5_6DeviceIS6_S7_EENS5_12Experimental14EmptyViewHooksENS5_12MemoryTraitsILj0EEEEEENS9_IPKiJSC_SE_SG_SI_EEESJ_SM_NS9_IPmJSC_SE_SG_SI_EEENS9_IPiJSC_SE_SG_SI_EEEEEvPT_T0_T1_T2_T3_T4_[_ZN12KokkosSparse12Experimental14spadd_symbolicIN13KokkosKernels12Experimental19KokkosKernelsHandleImidN6Kokkos6SerialENS5_9HostSpaceES7_EENS5_4ViewIPKmJNS5_10LayoutLeftENS5_6DeviceIS6_S7_EENS5_12Experimental14EmptyViewHooksENS5_12MemoryTraitsILj0EEEEEENS9_IPKiJSC_SE_SG_SI_EEESJ_SM_NS9_IPmJSC_SE_SG_SI_EEENS9_IPiJSC_SE_SG_SI_EEEEEvPT_T0_T1_T2_T3_T4_]+0x198): undefined reference to `KokkosSparse::Impl::SPADD_SYMBOLIC<KokkosKernels::Experimental::KokkosKernelsHandle<unsigned long const, int const, double const, Kokkos::Serial, Kokkos::HostSpace, Kokkos::HostSpace>, Kokkos::View<unsigned long const*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::MemoryTraits<1u> >, Kokkos::View<int const*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::MemoryTraits<1u> >, Kokkos::View<unsigned long const*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::MemoryTraits<1u> >, Kokkos::View<int const*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::MemoryTraits<1u> >, Kokkos::View<unsigned long*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::MemoryTraits<1u> >, false, false>::spadd_symbolic(KokkosKernels::Experimental::KokkosKernelsHandle<unsigned long const, int const, double const, Kokkos::Serial, Kokkos::HostSpace, Kokkos::HostSpace>*, Kokkos::View<unsigned long const*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::MemoryTraits<1u> >, Kokkos::View<int const*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::MemoryTraits<1u> >, Kokkos::View<unsigned long const*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::MemoryTraits<1u> >, Kokkos::View<int const*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::MemoryTraits<1u> >, Kokkos::View<unsigned long*, Kokkos::LayoutLeft, Kokkos::Device<Kokkos::Serial, Kokkos::HostSpace>, Kokkos::MemoryTraits<1u> >)'
collect2: error: ld returned 1 exit status
```

@brian-kelley can you investigate?

--- a/src/impl/generated_specializations_hpp/KokkosGraph_color_d1_eti_spec_avail.hpp.in
+++ b/src/impl/generated_specializations_hpp/KokkosGraph_color_d1_eti_spec_avail.hpp.in
@@ -1,5 +1,5 @@
-#ifndef KOKKOSSPARSE_SPADD_SYMBOLIC_ETI_SPEC_AVAIL_HPP_
-#define KOKKOSSPARSE_SPADD_SYMBOLIC_ETI_SPEC_AVAIL_HPP_
+#ifndef KOKKOSGRAPH_COLOR_D1_ETI_SPEC_AVAIL_HPP_
+#define KOKKOSGRAPH_COLOR_D1_ETI_SPEC_AVAIL_HPP_
 /*
 //@HEADER
 // ************************************************************************

--- a/src/impl/generated_specializations_hpp/KokkosGraph_color_d1_eti_spec_decl.hpp.in
+++ b/src/impl/generated_specializations_hpp/KokkosGraph_color_d1_eti_spec_decl.hpp.in
@@ -1,5 +1,5 @@
-#ifndef KOKKOSSPARSE_SPADD_SYMBOLIC_ETI_SPEC_DECL_HPP_
-#define KOKKOSSPARSE_SPADD_SYMBOLIC_ETI_SPEC_DECL_HPP_
+#ifndef KOKKOSGRAPH_COLOR_D1_ETI_SPEC_DECL_HPP_
+#define KOKKOSGRAPH_COLOR_D1_ETI_SPEC_DECL_HPP_
 /*
 //@HEADER
 // ************************************************************************

